cad/scripts/build_crank_arm.py

```
@@ -212,11 +219,24 @@ async def build(adapter) -> dict[str, str]:
 
-    # Fiducial dimple on the Z=0 face (which face carries it is arbitrary
-    # until assembly). Mid-plane cut of 2x depth: only the +Z half removes
-    # material, so the result is DIMPLE_DEPTH regardless of cut direction.
+    # HandleSeat = the z = ARM_THICKNESS face: the arm's FRONT face once
+    # placed (the composed Ry(180) turns local +z to machine -z, so this face
+    # looks south at the operator). 2026-09-02, ch11 p.14: the dimple and the
+    # keeper-ring anchor screw sit on THIS face, not the z = 0 one (which the
+    # earlier build carried the dimple on -- hidden against the chain wheel).
+    check(
+        f"create_plane HandleSeat (Front Plane, +{ARM_THICKNESS})",
+        await adapter.create_plane(CreatePlaneParameters(
```

Ruff (0.16.3)

[error] Undefined name `CreatePlaneParameters`

(F821)